src/numbers.js

```diff
@@ -1,45 +1,45 @@
 function add (a, b) {
-  // your code here
+  return (a+b)
```

Comment:
The logic in all of these is sound, but the formatting is off in all of them. You should have a clear space between each argument in the return statement and no brackets, so here:
```
  return a + b;
```